src/installer/corehost/cli/CMakeLists.txt

```diff
@@ -1,20 +1,14 @@
-if(RUNTIME_FLAVOR STREQUAL "Mono")
-    add_subdirectory(hostcommon)
-    add_subdirectory(fxr)
-    add_subdirectory(hostpolicy)
-else()
-    add_subdirectory(hostcommon)
-    add_subdirectory(apphost)
-    add_subdirectory(dotnet)
-    add_subdirectory(nethost)
-    add_subdirectory(test_fx_ver)
-    add_subdirectory(fxr)
-    add_subdirectory(hostpolicy)
+add_subdirectory(hostcommon)
+add_subdirectory(apphost)
+add_subdirectory(dotnet)
+add_subdirectory(nethost)
+add_subdirectory(test_fx_ver)
+add_subdirectory(fxr)
+add_subdirectory(hostpolicy)
 
-    add_subdirectory(test)
+add_subdirectory(test)
 
-    if(CLR_CMAKE_TARGET_WIN32)
-        add_subdirectory(comhost)
-        add_subdirectory(ijwhost)
-    endif()
-endif()
\ No newline at end of file
+if(CLR_CMAKE_TARGET_WIN32)
+    add_subdirectory(comhost)
+    add_subdirectory(ijwhost)
+endif()
```

Comment:
same here, I'm not totally sure what happens if we build apphost etc for both coreclr and mono for e.g. x64.